infrastructure/serve/cosmos.tf

```diff
@@ -0,0 +1,73 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+resource "azurerm_cosmosdb_account" "serve" {
+  name                = "cosmos-db-${var.naming_suffix}"
+  location            = var.core_rg_location
+  resource_group_name = var.core_rg_name
+  offer_type          = "Standard"
+  kind                = "GlobalDocumentDB"
+  # public_network_access_enabled = var.local_mode
+  tags = var.tags
+
+  capabilities {
+    name = "EnableServerless"
+  }
+
```

Comment:
Think we need an [identity block](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account#identity) to set System assigned identity?

Reply:
what do we need the cosmos MSI for? (thought it was only webapp->cosmos & webapp->sql we needed) or is it good practice to always assign an identity?

Reply:
For the web app <-> Cosmos, both need system assigned identities to communicate

Reply:
oh interesting! it worked okay in the [prototype](https://github.com/t-young31/FlowEHR/blob/62b60739765df01df51d2bba050a79e1e47391ea/infrastructure/serve/main.tf#L84) without a cosmos identity 🤔

Reply:
Hmm well I may be wrong! We can always add this in later if necessary then